packages/space-opera/src/components/model_viewer_snippet/components/open_button.ts

```diff
@@ -222,13 +220,13 @@ export class ImportCard extends LitElement {
     const dropdown = event.target as Dropdown;
     const value = dropdown.selectedItem?.getAttribute('value') || undefined;
     let snippet = '';
-    const simpleMap = {
+    const simpleMap: any = {
```

Comment:
In this modern era it's better to use `?? undefined` than `|| undefined`, or whatever other default value you want. Might want to do a quick code search to see where else `||` crops up.